src/features/jserrors/shared/cast-error.js

```diff
@@ -33,6 +33,13 @@ export function castError (error) {
 export function castPromiseRejectionEvent (promiseRejectionEvent) {
   let prefix = 'Unhandled Promise Rejection'
 
+  /**
+   * If the casted return value is falsy like this, it will get dropped and not produce an error event for harvest.
+   * We drop promise rejections that could not form a valid error stack or message deriving from the .reason attribute
+   * -- such as a manually invoked rejection without an argument -- since they lack reproduction value and create confusion.
+   * */
+  if (!promiseRejectionEvent.reason) return
```

Comment:
Can `promiseRejectionEvent` be undefined? Because the line right below this has a nullish accessor on it, so this is inconsistent rn.

Reply:
ty i'll take a look at that

Reply:
I have cleaned up the usage in there